examples/ScatterPlot.py

```diff
@@ -113,6 +114,38 @@ def createLabel(label, angle):
 w4.addItem(s4)
 s4.sigClicked.connect(clicked)
 
+# Enable hovering and show a tool tip hovered points
+s4.setAcceptHoverEvents(True)
+hoverPoints = []
```

Comment:
since we are doing `in` lookups, should this be a set instead of a list (same with `newPoints`?

Reply:
I did do that at first but used lists instead because I was slicing during the construction of the tool tip. The hovered points are returned in the reverse order in which they're drawn, so those drawn on top are prioritized for tool tip. That said, you're still right. Commit incoming.

Reply:
hmm....if order matters, maybe you can just use a `OrderedDict`, and just use the keys, and ignore the values? 

I don't have a feel at all if this is a performance bottleneck or not, so sticking with lists may be perfectly fine (I don't know how large those containers get so maybe the difference O(n) and O(1) may not be that different.

Maybe we should embed a `debug.Profiler` in this function so can do some benchmarking?  (I'm watching my two kids during the days which is why I haven't been able to experiment with this more. will try to mess with it more tonight).